codec-http/src/test/java/io/netty/handler/codec/http/cookie/ClientCookieEncoderTest.java

```diff
@@ -37,7 +37,9 @@ public void testEncodingMultipleClientCookies() {
         cookie2.setSecure(false);
         Cookie cookie3 = new DefaultCookie("myCookie3", "myValue3");
         String encodedCookie = ClientCookieEncoder.STRICT.encode(cookie, cookie2, cookie3);
-        assertEquals(c1 + c2 + c3, encodedCookie);
+        // Cookies should be sorted into decreasing order of path length, as per RFC6295.
+        // Where no path is provided, we assume maximum path length (so cookie3 comes first).
```

Comment:
I mean the comment should be "WHEN no path is provided", not "WHERE no path is provided"